### DISCUSSION TITLE
Achievements UI

### DISCUSSION THREAD

**Discussion by Ivan-Kalatchev** [community/community#30112] [Other Feature Feedback, Questions, & Ideas]
👋🏻 **Hi,**

Although I like the concept behind the new achievements feature 😄, the way they are presented really bugs me the wrong way... - Generally I think that they don't follow the GitHub Design aesthetic, maybe showing them like a list similar to that in the category selection dropdown in the discussion creation page will be more fitting with the general design guidelines. Thanks in advance!

**_Cheers,
Ivan Kalatchev_**

**Comment by radar-blip** [comment]
They will probably adjust accordingly as more achievements roll out. Maybe select only the ones you want to display also.
